doc/event-alarm-framework/event-alarm-framework.md

```diff
@@ -0,0 +1,1100 @@
+
+```
+sonic# event profile <profile-name>
+```
+The command takes name of specified file, validates it for its syntax and values; merges it with its internal static map of events *static_event_map*.
+
+#### 3.3.2.2 Configuration Commands
+```
+sonic(config)# logging server <ip> [log|event] 
+```
+Note: The 'logging server' command is an existing, already supported command. 
+It is only enhanced to take either 'log' or 'event' to indicate either native syslog messages or syslog messages corresponding to events alone are sent to the remote host.
+Support with VRF/source-interface and configuring remote-port are all backward comaptible and will be applicable to either 'log' or 'event' options.
+
+#### 3.3.2.3 Show Commands
+```
+sonic# show system health
+System Health: Normal, Warning or Error
+
+sonic# show event profile
+Event Profile: my_ev_prof.json
+
+sonic# show event [ detail | summary | severity <sev> | timestamp from <from-ts> to <to-ts> | recent <5min|1hr|1day> | sequence-id from <from-seq> to <to-seq> ] 
+
+'show event' commands would display all the records in EVENT table.
+
+sonic# show event 
+Sq No   State        Name                             Timestamp                   Description
+-----   --------     ------------------------------   --------------------------- --------------
+2292    -            TAM_SWITCH_ID_CHANGE             Wed Feb 10 18:08:27 2021    processSwitchTblEvent: Received new switch-id as 3        
+2291    raised       TEMPERATURE_EXCEEDED             Wed Feb 10 18:08:24 2021    temperatureCrossedThreshold: Current temperature of sensor/2 is 76 degrees
+182     cleared      TEMPERATURE_EXCEEDED             Wed Feb 10 09:52:12 2021    temperatureCrossedThreshold: Current temperature of sensor/2 is 76 degrees
+134     raised       TEMPERATURE_EXCEEDED             Wed Feb 10 09:32:01 2021    temperatureCrossedThreshold: Current temperature of sensor/2 is 76 degrees
+        
+sonic# show event details 
+Event History Details - 2292
+-------------------------------------------
+Sequence Number:   2292
+Severity:          informational
+Name:              TAM_SWITCH_ID_CHANGE
+Description:       processSwitchTblEvent: Received new switch-id as 3
+Timestamp:         Wed Feb 10 18:08:27 2021
+Source:            3
+State:             -
+-------------------------------------------
+Event History Details - 2291
+-------------------------------------------
+Sequence Number:   2291
+Severity:          critical
+Name:              TEMPERATURE_EXCEEDED
+Description:       temperatureCrossedThreshold: Current temperature of sensor/2 is 76 degrees
+Timestamp:         Wed Feb 10 18:08:24 2021
+Source:            sensor/2
+State:             raised
+
+sonic# show event summary
+
+Event Summary
+-------------------------------------------
+Raised:                 5
+Ack:                    1
+Cleared:                4
+Events:                 20
+-------------------------------------------
+
+sonic# show event severity critical
```

Comment:
Suggest to have severity column for all commands so even if you filter you will see the same look and feel.

Reply:
This would make the o/p to cross well above the 80-column limit.. The severity is anyway displayed with the details option.